### DISCUSSION TITLE
Volume can be attached/detached from UI, but Kubernetes Pod/StatefulSet etc cannot use it

### DISCUSSION THREAD

**Discussion by jerryTJ** [longhorn/longhorn#3534] [Q&A]
Longhorn:v1.2.3
K8s: v1.20.0
File System: xfs
Mount Propagate：enabled
Error info：attach detach-controller AttacheVolume.Attach failed for volume "pvc-xxx"
 rpc error: code=DeadlineExceeded desc=volume "pvc-xxx"  failed to attach to node “xxx”

Can anyone help me？

**Comment by jerryTJ** [comment]
<img width="1427" alt="image" src="https://user-images.githubusercontent.com/5989442/149475232-3f75863b-680b-4199-9607-f5e559c0b9eb.png">

**Reply by innobead** [reply]
cc @PhanLe1010

**Reply by PhanLe1010** [reply]
Is it a RWX volume or RWO volume.
Did you make sure that all prerequisites are met https://longhorn.io/docs/1.2.3/deploy/install/? RWX volumes need the nfsv4 client on the hosts

After checking those conditions, if you still have the problem, please upload a support bundle here. I will look into the volume

**Reply by jerryTJ** [reply]
RWO volume  and the result of env check:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/5989442/150117742-2f60184e-baf4-4e07-9af6-87b1eff5d961.png">

**Reply by PhanLe1010** [reply]
Can you provide:
* kubectl describe pvc <PVC-NAME>
* kubectl get volumes.longhorn.io pvc-e453... -n longhorn-system

**Reply by jerryTJ** [reply]
I recreated other one，Is it caused by the network failure？
![image](https://user-images.githubusercontent.com/5989442/150323171-41d6516b-6f0e-4552-a2fe-0d919b8a0ca4.png)

**Reply by PhanLe1010** [reply]
volume robustness is `unknown`. We need more data to find out why. Can you provide us with a support bundle?

**Reply by jerryTJ** [reply]
Sorry for taking so long to reply to you.
[longhorn-support-bundle_c42d4bc4-40d2-4c75-82c3-a38ba8916f1c_2022-01-22T08-21-40Z.zip](https://github.com/longhorn/longhorn/files/8024429/longhorn-support-bundle_c42d4bc4-40d2-4c75-82c3-a38ba8916f1c_2022-01-22T08-21-40Z.zip)
.
